MyOwnORM/MyOwnORM/CustomDbSetReflection.cs

```diff
@@ -0,0 +1,823 @@
+﻿using Microsoft.CSharp.RuntimeBinder;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Reflection;
+using System.Reflection.PortableExecutable;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MyOwnORM
+{
+    public class CustomDbSetReflection<T> where T : class
```

Comment:
since this class works with reflection separate out of here all methods where you build strings in another service

Reply:
Fixed